src/wp-includes/class-wp-http-streams.php

```diff
@@ -101,8 +101,9 @@ public function request( $url, $args = array() ) {
 			 * @since 2.8.0
 			 * @since 5.1.0 The `$url` parameter was added.
 			 *
-			 * @param bool   $ssl_verify Whether to verify the SSL connection. Default true.
-			 * @param string $url        The request URL.
+			 * @param bool|string $ssl_verify Boolean to control whether to verify the SSL connection
+			 *                                or path to an SSL certificate.
+			 * @param ?string     $url        Optional. The request URL.
```

Comment:
Can you clarify whey you use `?` before string?

Reply:
I can switch the `$url` parameter's type to `string|null`, if it's preferable.

I marked it as explicitly nullable if ever an implementation that uses these hooks passes `null` as a second parameter instead of omitting it.

There are similar occurrences of using `?` in the codebase.

Reply:
> I marked it as explicitly nullable if ever an implementation that uses these hooks passes `null` as a second parameter instead of omitting it.

I can see the idea behind this, but the same applies to every parameter of every filter and action in WordPress. This is better address in tests and tooling rather than nulling the parameter.